src/iframe-placeholder.ts

```diff
@@ -1,33 +0,0 @@
-// This file is just a placeholder with some pseudo-code for the injected code on ArConnect Embedded. That is, the code
```

Comment:
Added again in https://github.com/arconnectio/ArConnect/pull/555/files#diff-e752dd3e9111991e7006ebd8769b49e4c002226e0a19ca0a1bca0de3788a0598 as it was probably merged to `development` by mistake, but it doesn't really matter.